### PR TITLE
feat(ui) - add tooltip for RadioButtonGroup UI component [LW-9983]

### PR DESCRIPTION
**Pull request by tommayeliog**
# Checklist

- [x] JIRA - https://input-output.atlassian.net/browse/LW-9983<link>
- [x] Proper tests implemented
- [x] Screenshots added.

---

## Proposed solution
With this PR I'm adding tooltip option for RadioGroupGroup's options.

## Testing
Storybook

## Screenshots
Screenshot 2024-03-14 at 11 04 38
Screenshot 2024-03-14 at 11 04 26
Screenshot 2024-03-14 at 11 04 10

### CODE REVIEW

File: packages/ui/src/design-system/radio-button/radio-button.component.tsx [hunk 1]
```diff
@@ -72,16 +59,44 @@ export const RadioButtonGroup = ({
...
+                  <RadixRadioGroup.Indicator
+                    className={cx.radioGroupIndicator}
+                  />
+                </RadixRadioGroup.Item>
+                {!!tooltipText && (
+                  <Tooltip label={tooltipText} delayDuration={600}>
+                    {labelContent}
+                  </Tooltip>
+                )}
+                {!tooltipText && labelContent}
```
Comment (przemyslaw-wlodek) [inline packages/ui/src/design-system/radio-button/radio-button.component.tsx]:
```suggestion
                {tooltipText ? (
                  <Tooltip label={tooltipText} delayDuration={600}>
                    {labelContent}
                  </Tooltip>
                ) : labelContent}
```

**Review by przemyslaw-wlodek — APPROVED** [review]
Good! Almost all comments were resolved on call 🎉

File: packages/ui/src/design-system/radio-button/radio-button.component.tsx (hunk 1 above)
Comment (tommayeliog) [reply packages/ui/src/design-system/radio-button/radio-button.component.tsx]: thanks, done!

File: packages/ui/src/design-system/radio-button/radio-button.component.tsx [hunk 2]
```diff
@@ -80,12 +83,13 @@ export const RadioButtonGroup = ({
                     className={cx.radioGroupIndicator}
                   />
                 </RadixRadioGroup.Item>
-                {!!tooltipText && (
-                  <Tooltip label={tooltipText} delayDuration={600}>
+                {tooltipText ? (
+                  <Tooltip label={tooltipText} delayDuration={800}>
```
Comment (przemyslaw-wlodek) [inline packages/ui/src/design-system/radio-button/radio-button.component.tsx:87]: Is it agreed with Dan / Allan?

File: packages/ui/src/design-system/radio-button/radio-button.css.ts [hunk 3]
```diff
@@ -52,50 +50,46 @@ export const radioGroupItem = style([
     },
 
     selectors: {
+      // '&[data-state=unchecked]': {
+      //   border: 'none',
+      //   outline: `none`,
+      // },
```
Comment (przemyslaw-wlodek) [inline packages/ui/src/design-system/radio-button/radio-button.css.ts:56]: leftover

File: packages/ui/src/design-system/radio-button/radio-button.component.tsx [hunk 4]
```diff
@@ -34,6 +34,7 @@ export const RadioButtonGroup = ({
   ...props
 }: Readonly<RadioButtonGroupProps>): JSX.Element => {
   const hasIcon = options.some(({ icon }) => Boolean(icon));
+  console.log({ hasIcon });
```
Comment (przemyslaw-wlodek) [inline packages/ui/src/design-system/radio-button/radio-button.component.tsx:37]: console.log

Comment (tommayeliog) [reply packages/ui/src/design-system/radio-button/radio-button.component.tsx]: removed

File: packages/ui/src/design-system/radio-button/radio-button.component.tsx (hunk 2 above)
Comment (tommayeliog) [reply packages/ui/src/design-system/radio-button/radio-button.component.tsx:86]: yes, Dan is ok with that

File: packages/ui/src/design-system/radio-button/radio-button.css.ts (hunk 3 above)
Comment (tommayeliog) [reply packages/ui/src/design-system/radio-button/radio-button.css.ts]: removed, thanks

File: packages/ui/src/design-system/radio-button/radio-button.component.tsx [hunk 5]
```diff
@@ -3,8 +3,10 @@ import React from 'react';
 import * as RadixRadioGroup from '@radix-ui/react-radio-group';
 import cn from 'classnames';
 
+import { Button } from '..';
```
Comment (wklos-iohk) [inline packages/ui/src/design-system/radio-button/radio-button.component.tsx:6]: unused import

File: packages/ui/src/design-system/radio-button/radio-button.stories.tsx [hunk 6]
```diff
@@ -28,43 +29,54 @@ export default {
...
       id: 'active',
       onValueChange: (): undefined => undefined,
     },
     {
-      options: getOptions(label),
+      options: getOptions(1, label, icon),
       disabled: true,
-      onValueChange: (): undefined => undefined,
+      onValueChange: () => {
+        console.log('test');
```
Comment (przemyslaw-wlodek) [inline packages/ui/src/design-system/radio-button/radio-button.stories.tsx:75]: Please use SB actions instead

Comment (tommayeliog) [reply packages/ui/src/design-system/radio-button/radio-button.stories.tsx:75]: removed

File: packages/ui/src/design-system/radio-button/radio-button.component.tsx (hunk 5 above)
Comment (tommayeliog) [reply packages/ui/src/design-system/radio-button/radio-button.component.tsx:6]: removed

**Pull request merged**